src/component/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "react-router-dom";
+import "./sidebar.css";
+
+const Sidebar = () => {
+  const activeStyle = ({ isActive }) => {
+    return isActive
+      ? "active-link md-pd aside-heading  "
+      : "inactive-link md-pd aside-heading  ";
+  };
+
```

Comment:
Appreciation : Nice that you did this with active link

Reply:
thanks bipul